docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Planning IAM integration strategy
+menuTitle: IAM integration
+description: Considerations and .
```

Comment:
Looks like the end of the sentence is missing?

Reply:
🤦 My bad. I have fixed this.